docs/src/main/paradox/ftp.md

```diff
@@ -0,0 +1,96 @@
+# FTP Connector
+
+The FTP connector provides Akka Stream sources to connect to FTP, FTPs and SFTP servers. Currently, two kinds of sources are provided:
+
+* one for browsing or traversing the server recursively and, 
+* another for retrieving files as a stream of bytes.
+
+## Artifacts
+
+sbt
+:   @@@vars
+    ```scala
+    libraryDependencies += "com.typesafe.akka" %% "akka-stream-alpakka-ftp" % "$version$"
+    ```
+    @@@
+
+Maven
+:   @@@vars
+    ```xml
+    <dependency>
+      <groupId>com.typesafe.akka</groupId>
+      <artifactId>akka-stream-alpakka-ftp_$scala.binaryVersion$</artifactId>
+      <version>$version$</version>
+    </dependency>
+    ```
+    @@@
+
+Gradle
+:   @@@vars
+    ```gradle
+    dependencies {
+      compile group: "com.typesafe.akka", name: "akka-stream-alpakka-ftp_$scala.binaryVersion$", version: "$version$"
+    }
+    ```
+    @@@
+
+## Usage
+
+### Configuring the connection settings
+
+In order to establish a connection with the remote server, you need to provide an specialized version of a @scaladoc[RemoteFileSettings](akka.stream.alpakka.ftp.RemoteFileSettings) instance. It's specialized as it depends on the kind of server you're connecting to: FTP, FTPs or SFTP.
+
+Scala
+: @@snip (../../../../ftp/src/test/scala/akka/stream/alpakka/ftp/BaseFtpSpec.scala) { #create-settings }
+
+Java
+: @@snip (../../../../ftp/src/test/java/akka/stream/alpakka/ftp/FtpSourceTest.java) { #create-settings }
+
+The configuration above will create an anonymous connection with a remote FTP server in passive mode. For both FTPs and SFTP servers, you will need to provide the specialized versions of these settings: @scaladoc[FtpsSettings](akka.stream.alpakka.ftp.RemoteFileSettings.FtpsSettings) or @scaladoc[SFtpSettings](akka.stream.alpakka.ftp.RemoteFileSettings.SFtpSettings) 
```

Comment:
This does not properly link. Use `akka.stream.alpakka.ftp.RemoteFileSettings$$FtpsSettings` and `akka.stream.alpakka.ftp.RemoteFileSettings$$SFtpSettings`